python/cloudtik/providers/_private/huaweicloud/config.py

```diff
@@ -1077,10 +1079,12 @@ def get_huaweicloud_workspace_info(config):
     if bucket_name:
         resp = obs_client.getBucketLocation(bucket_name)
         if resp.status < 300:
-            managed_cloud_storage_uri = resp.body.location
+            location = resp.body.location
+            _storage_uri = HWC_OBS_BUCKET_LOCATION.format(
+                bucket_name=bucket_name, location=location)
             managed_cloud_storage = {
                 HWC_MANAGED_CLOUD_STORAGE_OBS_BUCKET: bucket_name,
-                CLOUDTIK_MANAGED_CLOUD_STORAGE_URI: managed_cloud_storage_uri}
+                CLOUDTIK_MANAGED_CLOUD_STORAGE_URI: _storage_uri}
```

Comment:
CloudTik (Spark) data access to cloud storage are basically through Hadoop File System. So we originally target the cloud storage URI as Hadoop file system uri to access the bucket. for example:
for AWS S3, its s3a://bucket_name, for GCS: gcs://bucket_name, and so on.

This information is shown as workspace info which is useful for user to access the data in their jobs using this uri.
It's not the URI that in the concept of cloud provider.

For Huawei Cloud, Hadoop file system URI to access may look like: obs://bucket_name or something.

Reply:
So that's it, thank you